### PR TITLE
Sprint 6: Find an intervention tweaks

### PR DESCRIPTION
**Pull request by Gweaton**
## Changes in this PR
- Remove browse by need and category links from Find
- Fix broken browse by region links after directory moved
- Correct "selected" implementation on Find multi select
- Change "shortlisted" to "selected" throughout

### CODE REVIEW

File: app/views/sprint-6/book-and-manage/make-a-referral/find-an-intervention/find-multi-select.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1 class="govuk-heading-xl">Find an intervention - Alex River</h1>
         <div class="govuk-form-group">
           <label for="needs" class="govuk-label">Criminogenic needs</label>
           <select id="needs" name="needs" class="govuk-select" multiple="multiple" data-auto-complete="true">
-            <option value="accommodation" selected>Accommodation</option>
+            <option value="accommodation" selected="selected">Accommodation</option>
```
Comment (lawrence-forooghian) [inline app/views/sprint-6/book-and-manage/make-a-referral/find-an-intervention/find-multi-select.html:50]: I'm pretty sure this is valid as it is - according to the spec `selected` is a boolean attribute, meaning its presence switches it on: https://html.spec.whatwg.org/multipage/form-elements.html#attr-option-selected

**Review by lawrence-forooghian — APPROVED** [review]

File: app/views/sprint-6/book-and-manage/make-a-referral/find-an-intervention/find-multi-select.html (hunk 1 above)
Comment (Gweaton) [reply app/views/sprint-6/book-and-manage/make-a-referral/find-an-intervention/find-multi-select.html:50]: You're right. I should have spent more time looking into it, it's weird the linter just does that... I'm gonna remove the commit

Comment (lawrence-forooghian) [reply app/views/sprint-6/book-and-manage/make-a-referral/find-an-intervention/find-multi-select.html]: What linter was it?

Comment (Gweaton) [reply app/views/sprint-6/book-and-manage/make-a-referral/find-an-intervention/find-multi-select.html]: Prettier (vs code add-on)

**Pull request merged**